app/error.js

```
@@ -0,0 +1,40 @@
+        <h1 className="text-3xl font-bold text-red-600 mb-4">Something went wrong!</h1>
+        
+        <div className="mb-6 text-gray-700">
+          <p>We're sorry, but there was an error with your request.</p>
+          <p className="mt-2 text-sm text-gray-500">Error details: {error?.message || 'Unknown error'}</p>
+        </div>
+        
+        <div className="flex flex-col gap-4">
+          <button
+            onClick={() => reset()}
+            className="px-4 py-2 bg-blue-600 text-white rounded hover:bg-blue-700 transition-colors"
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)